pages/partials/navbar.ejs

```diff
@@ -124,3 +125,18 @@ if (typeof authz_data !== 'undefined' && authz_data.has_instructor_view) {
 </ul>
 </div>
 </nav>
+<% // Mapping navPage to navtab sets
+const navPagesTabs = {
+    instance_admin: 'instructorInstanceAdminTabs',
+    course_admin: 'instructorCourseAdminTabs',
+    assessment: 'instructorAssessmentTabs',
+    question: 'instructorQuestionTabs'
```

Comment:
Can we rename all these files so they have a common prefix? E.g., `navTabsInstructorInstanceAdmin`, `navTabsInstructorQuestion`, etc. I find this makes the directory listings easier to navigate because all the "navTab" file are next to each other.

To bikeshed the naming scheme a bit: are we calling these `tabs`, `navtabs`, or `navTabs`? I don't have a strong opinion, but consistency would be good.

Reply:
> To bikeshed the naming scheme a bit: are we calling these tabs, navtabs, or navTabs?

Bootstrap calls them `.nav-tabs`, so maybe `navTabs*`?

Reply:
I can get that into a follow up refactor

Reply:
Can you just do it now? I'm hesitant to merge code that I know needs immediate cosmetic improvements.

Reply:
Sure thing. What would you think about putting them inside a folder navTabs instead of the common prefix?

Reply:
Sounds good! I think there were also internal variable names that should be changed to match? `navtab` -> `navTab`? I haven't thought about them all other than the desire for consistency 😄 

In cases where we use a mix of kebab-case, snake_case, and camelCase, we try and keep the word boundaries the same. So `instance_questions`, `instanceQuestions`, `instance-questions`, for example.

Reply:
Done!